examples/federated-learning/tff/nvflare/base-client/base-client.yaml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    checkov.io/skip1: CKV_K8S_21=Default namespace used
+    checkov.io/skip2: CKV_K8S_43=Image version used instead of digest
+    checkov.io/skip3: CKV_K8S_14=Image name and tag is being replaced by kustomize
+    checkov.io/skip5: CKV_K8S_8=Liveness probes not set
+    checkov.io/skip9: CKV_K8S_40=No high UID
+    checkov.io/skip10: CKV_K8S_22=No read-only filesystem
+    checkov.io/skip12: CKV_K8S_38=No service account token
+    checkov.io/skip14: CKV_K8S_9=Readiness probes not set
+    checkov.io/skip16: CKV_K8S_31=No seccomp profile
```

Comment:
Here you are explaining what the check does, but you're not explaining why we don't fix these issues. I would suggest fixing these issues.

You probably want to skip `CKV_K8S_43` and `CKV_K8S_14` because we handle that using Kustomize.

Reply:
Also, remove the ones that you might have fixed from the list of skipped checks.

Reply:
Done